spec/models/conference_spec.rb

```diff
@@ -41,12 +41,21 @@
       end
     end
   end
+  
+  describe '#dates' do
+    subject { FactoryGirl.build_stubbed(:conference) }
+    
+    xit 'has a date range' do
+      expect(subject.date_range).to be_an_instance_of DateRange
+    end
+  end
```

Comment:
The new `date_range` method returns a formatted string, so now it is not a DateRange anymore. 
How about: 
`expect(subject.date_range).to be`